gems/aws-sdk-s3/spec/object/presigned_url_spec.rb

```diff
@@ -159,14 +159,14 @@ module S3
           allow(Time).to receive(:now).and_return(now)
           url, headers = obj.presigned_request(
             :get, expires_in: 86_400, request_payer: 'peccy')
-          expect(url).to eq(
+          expect(CGI.parse(url)).to eq(CGI.parse(
```

Comment:
Why do we need CGI parsing now?

Reply:
The order of the query parameters may differ.